resources/assets/uad_lists.json

```
@@ -5582,7 +5582,7 @@
   {
     "id": "com.samsung.safetyinformation",
     "list": "Oem",
-    "description": "Some safety information telling you not to put your phone in your eyes (it's not a joke)\n",
+    "description": "Safety information telling you not to put your phone in your eyes or other dangerous stuff\n",
```

```suggestion
    "description": "Safety information telling you not to put your phone in your eyes or other dangerous stuff\n",
```
This kind of sounds weird. You don't put your phone *in* your eyes. Does it really say that?😅

---

I haven't opened the app myself, so no clue about that. I feel it would the old regular legal stuff, so I won't be surprised if it actually has that exact statement.

---

Hmm, honestly no idea what we should put. What Samsung says makes no sense to me.

---

I just made a change. Have a look.

---

Add "your phone" instead of just "phone".

---

Done